worldguard-bukkit/src/main/java/com/sk89q/worldguard/bukkit/listener/WorldGuardEntityListener.java

```diff
@@ -851,4 +852,27 @@ private boolean checkItemFrameProtection(Entity attacker, ItemFrame defender) {
         return false;
     }
 
+    /**
+     * Checks regions and config settings to protect items from being knocked
+     * out of armor stand.
+     * @param attacker attacking entity
+     * @param defender armor stand being damaged
+     * @return true if the event should be cancelled
+     */
+    private boolean checkArmorStandProtection(Entity attacker, ArmorStand defender) {
```

Comment:
You can generalize the `checkItemFrameProctection` method and pass a flag to that method.

That way you don't need double methods.